framework_laptop.c

```diff
@@ -750,7 +750,7 @@ static int framework_probe(struct platform_device *pdev)
 	return ret;
 }
 
-static int framework_remove(struct platform_device *pdev)
+static void framework_remove(struct platform_device *pdev)
```

Comment:
This breaks compilation for older kernels.
There should be a version check.

Reply:
For which kernel versions did you experience breakage ?

Reply:
oh I get what you mean, yeah I agree, having a version check is the way to go